docs/DeveloperGuide.md

```diff
@@ -80,9 +80,9 @@ The `UI` component,
 **API** :
 [`Logic.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/logic/Logic.java)
```

Comment:
I think the link should be updated to our group repo

Reply:
oh yea nice spot 👍🏻